### DISCUSSION TITLE
Emojis for icon

### DISCUSSION THREAD

**Discussion by aam1r** [outline/outline#1851] [Ideas]
I feel like the current set of icons is quite limiting:

![image](https://user-images.githubusercontent.com/566418/106206185-a5826000-618d-11eb-90bf-1e973f1fdb92.png)

Instead of having to design more icons, it would be cool if we could instead enter an emoji as an icon.

**Comment by aam1r** [comment]
Related to https://github.com/outline/outline/pull/177

**Comment by tommoor** [comment]
We're happy to design more icons if you have any specific requests for new ones.

I think that over-usage of emoji's makes an application look a little cheap and "samey" to be honest, I'd rather maintain consistent iconography in a distinctive style even if that means slightly less choice and no poop emoji 😆

**Reply by dkkb** [reply]
I think it could be a good idea to allow users to input custom images as collection icons (😆).

e.g.

<img src="https://user-images.githubusercontent.com/82504881/141741015-51d55a5e-8803-4a3a-bc07-b6f0b91b5fe0.png" width=100>

**Reply by Limezy** [reply]
And what about integrating something like https://github.com/FortAwesome/Font-Awesome which is precisely meant to offer a consistant package of icons with a high quality standard ?
I think this is what is used by Flarum forum app as their icon provider.

**Reply by almereyda** [reply]
You may as well want to consider Fork Awesome instead :) https://github.com/atisawd/boxicons

Yet there is no lack of possible icon sources. It is rather a question of style and choice, to decide which ones to integrate.

Icon aggregators

* https://github.com/ryanoasis/nerd-fonts
  * https://www.nerdfonts.com/
* https://github.com/iconify/iconify
  * https://iconify.design/

Icon collections

* https://github.com/atisawd/boxicons
  * https://boxicons.com/
* https://github.com/stephenhutchings/typicons.font
  * https://www.s-ings.com/typicons/
    * https://www.s-ings.com/projects/microns-icon-font/
* https://github.com/tabler/tabler-icons
  * https://tabler-icons.io/
* https://github.com/icons8/webicon
  * https://blog.icons8.com/articles/webicon-svg-sets-icons/
* https://github.com/mikolajdobrucki/ikonate
  * https://ikonate.com/
* https://github.com/feathericons/feather
  * https://feathericons.com/
* https://github.com/evil-icons/evil-icons
  * https://evil-icons.io/
* https://github.com/michaelampr/jam
  * https://jam-icons.com/
* https://github.com/antfu/unplugin-icons
  * https://www.npmjs.com/package/unplugin-icons

Framework icons

* https://github.com/carbon-design-system/carbon-icons-svelte
  * https://carbon-icons-svelte.onrender.com/
* https://github.com/ionic-team/ionicons
  * https://ionic.io/ionicons
* https://github.com/tailwindlabs/heroicons
  * https://heroicons.com/
* https://github.com/primer/octicons
  * https://primer.style/octicons/

Brand icons

* https://github.com/simple-icons/simple-icons
  * https://simpleicons.org/
* https://github.com/devicons/devicon
  * https://devicon.dev/

Did I understand correctly that the icons visually represent the Outline brand, and are therefore hand-crafted?

**Reply by tommoor** [reply]
I don't expect anyone to recognize the Outline icons necessarily but it's important that the iconography be consistent in its style and weight ☺️

If there is an iconset that fits the style well, and is appropriately licensed, we should seriously consider including it

**Reply by cdalvaro** [reply]
This would be great!

**Comment by tommoor** [comment]
It is now possible to use both icon and emoji for collections and documents in https://github.com/outline/outline/releases/tag/v0.78.0
